src/models/msg/transfer/msg_transfer.ts

```diff
@@ -0,0 +1,25 @@
+import { Categories } from '../types';
+
+class MsgTransfer {
+    public category: Categories;
+    public type: string;
+    public signer: string;
+    public json: any;
+
+    constructor(payload: any) {
+      this.category = 'transfer';
+      this.type = payload.type;
+      this.signer = payload.signer;
+      this.json = payload.json;
+    }
+
+    static fromJson(json: any) {
+      return new MsgTransfer({
+        json,
+        type: json['@type'],
+        signer: json.signer,
+      });
+    }
+}
+
+export default MsgTransfer;
```

Comment:
I haven't checked the other messages but this specific tx msg `1BCD30C1740E7949D68B9383C0865CD04656599B2045B75A5B27CEA34FA627C0` crashed the dev server. It displays correctly if i display it as unknown.

Reply:
I see, I updated MsgTransfer types and tested it. It displays the information correctly now.